content/attacks/posts/2023-01-12-Lendhub.md

```diff
@@ -0,0 +1,32 @@
+---
+date: 2023-01-12
+target-entities: Lendhub
+entity-types: Defi
+attack-types: Smart Contract Exploit
+title: "Lendhub Hacked for $6 Million"
+---
+
+## Summary
+
+A hacker exploited a vulnerability in the LendHub protocol to steal approximately $6 million in digital assets. The vulnerability was caused by the existence of two IBSV tokens on the platform, one of which had been deprecated but not removed. The attacker was able to mint and redeem tokens in the old market while borrowing against them in the new market, ultimately making off with the majority of the assets on the platform.
+
+## Attackers
+
+The attackers behind the Lendhub hack remain unidentified.
+
+- Lendhub Hacker wallet: [0x9d0163e76bbcf776001e639d65f573949a53ab03](https://etherscan.io/address/0x9d0163e76bbcf776001e639d65f573949a53ab03)
+
+## Losses
+
+Lendhub lost approximately $6 million.
+
+## Timeline
+
+- **January 12, 2023:**  Lendhub is hacked
```

Comment:
Provide malicious transaction with exact time if possible, the sentence looks incomplete.